src/reference/00-Getting-Started/07A-Scopes.md

```diff
@@ -393,3 +375,178 @@ lazy val util = (project in file("util"))
     // other settings
   )
 ```
+
+#### Delegation in detail (.value lookup)
+
+Now that we've covered all the details of scoping, we can explain the `.value`
+lookup in detail. It's ok to skip this section at in the beginning.
+To summarize what we've learned so far:
+
+- A particular scope consists of three scope axes: Subprojects, Dependency configuration, and Tasks.
+  You can think of axes like as digits in a number, or (r, g, b) components in a color.
+- There's a special value `Global` scope (written as `*` in shell) that can be applied to either of the axes.
+- There's a special value `ThisBuild` scope (written as `{.}` in shell) only for Subprojects axis.
+- build.sbt can reference a scoped key using `.in(...)` method.
+
+Now let's suppose we have a build definition as follows:
+
+```scala
+lazy val foo = settingKey[Int]("")
+lazy val bar = settingKey[Int]("")
+
+lazy val projA = (project in file("a"))
+  .settings(
+    foo := {
+      (bar in Test).value + 1
+    },
+    bar in Compile := 1
+  )
+```
+
+Inside of `foo`'s setting body the dependency a scoped key `(bar in Test)` is declared.
+However, as you see, `bar in Test` is undefined in `projA`.
+sbt is able to still resolve `(bar in Test)` to another scoped key,
+and initialize `foo` as `2`.
```

Comment:
I know it sounds obvious, but it would be a good idea to explain that this is because (if I understand correctly) the `Test` scope `extend`s the `Compile` scope. It might also be worth explaining what it means for a scope to extend another scope.